.github/workflows/build.yml

```diff
@@ -132,7 +132,7 @@ jobs:
     - name: Install dependencies
       run: |
         brew install gettext awscli
-        echo "::set-env name=PATH::/usr/local/opt/gettext/bin:$PATH"
+        echo >>$GITHUB_PATH /usr/local/opt/gettext/bin
```

Comment:
not important at all, though the github doc https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
use the syntax that put ` >> $GITHUB_ENV` at the end 

```
echo "action_state=yellow" >> $GITHUB_ENV
echo "/path/to/dir" >> $GITHUB_PATH
```

Out of curiosity, why would you change the order, or maybe they update the doc recently ?

Reply:
right, it makes lots of sense :+1: